charts/vela-core/values.yaml

```diff
@@ -62,6 +62,9 @@ affinity: {}
 rbac:
   create: true
 
+controllerArgs:
+  reSyncPeriod: 5m
```

Comment:
can we make it more fast? such as 2m ? how did we get the time result?

Reply:
2m is a bit aggressive I think. For example, with 2c4g the standard application supported in 1 minute is 200~300 (without advanced optimization). If the reSyncPeriod is set to 2m, it means with 2c4g we can support 400~600 applications. Therefore, for the default helm install configuration (0.5c1g), we can only support 100~150 simple applications. For 5m, we can have 250+ applications.

Reply:
This data should be documented well.